edu-services/gradebook-service/hibernate/src/java/org/sakaiproject/tool/gradebook/GradingEvent.java

```diff
@@ -37,6 +38,7 @@
     private GradableObject gradableObject;
     private String grade;
     private Date dateGraded;
+    private int excluded;
```

Comment:
With hibernate its best to use Objects that better map to the state of the database, so Integer makes more sense here.